src/main.rs

```diff
@@ -192,107 +227,150 @@ impl<'a> Interpreter<'a> {
                 ArrayGet => self.do_array_get(),
 
                 Call(ref target) => {
-                    let new_func = self.module.funcs.get(target).unwrap();
-                    let mut frame = CallFrame::for_fn(new_func, (func, pc));
+                    let new_func = self.module.funcs[target].clone();
```

Comment:
I guess I was expecting inlining to happen here somewhere. Maybe it happens later.